app/main.js

```diff
@@ -4,10 +4,12 @@ let mainWindow = null;
 
 app.on('ready', () => {
   mainWindow = new BrowserWindow({
-    width: 300,
-    height: 500,
-    maxWidth: 800,
-    maxHeight: 600
+    width: 1000,
+    height: 600,
+    // width: 300,
+    // height: 500,
+    // maxWidth: 800,
+    // maxHeight: 600
```

Comment:
This is a temp fix in order to work in full screen when we are in developer mode.